Frontend/src/Components/BatteryCells/DataPack.js

```diff
@@ -0,0 +1,25 @@
+import { Box, Text, VStack, Flex, Spacer } from "@chakra-ui/react";
+
+export default function DataPack(props) {
+    return (
+        <VStack
+            borderWidth={1}
+            borderColor='black'
+            spacing={0}
+            bg={props.bg}
+        >
+            <Flex w='95%'>
+                <Text fontSize='xs'>
+                    {props.dataTitle}:
+                </Text>
+                <Spacer />
+                <Text fontSize='xs'>
+                    {props.dataValue.toFixed(3)}{props.dataUnit}
+                </Text>
+            </Flex>
+            <Box border='1px' w='15vw' h='0.7vh'>
+                {props.battery}
+            </Box>
```

Comment:
```suggestion
```
The nominal range boxes can be removed if they cause the view to overflow. Otherwise, if you have any ideas for fitting all of the data within the container without decreasing font/readability, that would be awesome.